### ISSUE TITLE
feathers2-neo webpage - link to micropython .bin file causes HTML 404 error

### ISSUE THREAD

**Issue by PaulskPt** [UnexpectedMaker/feathers2#1]
Hi UM, I ordered a neo with Pimoroni. Expecting delivery today. 
I was reading your webpage: https://unexpectedmaker.com/feathers2-neo. 
Tried to use the link: 
https://micropython.org/resources/firmware/featherS2neo-20211025-unstable-v1.17-99-g99221cd11.bin 
but received a HTML 404 error. Error is probably caused because the file named has been replaced?
I went to https://micropython.org. Searched and found the feathers2-neo. Downloaded the latest 'nightly build'.
Perhaps it is better to use the more general link: https://micropython.org/download/featherS2neo/ on your feathers2-neo webpage.
Greetings,
Paulus

**Comment by mcauser** [comment]
Yep, the page should be updated to link to the new downloads page: https://micropython.org/download/featherS2neo/

**Comment by PaulskPt** [comment]
Tnx
